src/Integration.UnitTests/ProfileConflicts/ConflictsManagerTests.cs

```diff
@@ -79,16 +81,16 @@ public void TestInit()
             this.dte = new DTEMock();
             this.projectHelper.CurrentActiveSolution = new SolutionMock(dte);
 
-            this.testSubject = new ConflictsManager(serviceProvider, new SonarLintOutputLogger(serviceProvider), new ProjectBinderFactory(), fileSystem);
+            this.testSubject = new ConflictsManager(serviceProvider, new SonarLintOutputLogger(serviceProvider), new ProjectBinderFactory(serviceProvider, fileSystem), fileSystem);
         }
 
         [TestMethod]
         public void ConflictsManager_Ctor()
         {
-            Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(null, new Mock<ILogger>().Object, new ProjectBinderFactory(), fileSystem));
-            Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(serviceProvider, null, new ProjectBinderFactory(), fileSystem));
+            Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(null, new Mock<ILogger>().Object, new ProjectBinderFactory(serviceProvider), fileSystem));
+            Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(serviceProvider, null, new ProjectBinderFactory(serviceProvider), fileSystem));
             Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(serviceProvider, new Mock<ILogger>().Object,null, fileSystem));
-            Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(serviceProvider, new Mock<ILogger>().Object,new ProjectBinderFactory(), null));
+            Exceptions.Expect<ArgumentNullException>(() => new ConflictsManager(serviceProvider, new Mock<ILogger>().Object,new ProjectBinderFactory(serviceProvider), null));
```

Comment:
I know you mentioned re-writing the conflicts manager and some other tests later, but as you continue refactoring look out for tests that you could fix up immediately e.g. it would have been as easy to make this one use a mock of `IProjectBindingFactory` and then it might not need to change again.